docs/event-bus/03-04-knative-eventing-mesh.md

```diff
@@ -41,3 +41,76 @@ To reach the new Eventing Mesh, use an HTTP request with the `/events` path.
+The default channel implementation can be changed during installation using an installation-override like this:
+
+```bash
+$ cat << EOF | kubectl apply -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: knative-eventing-overrides
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    component: knative-eventing
+    kyma-project.io/installation: ""
+data:
+  knative-eventing.channel.default.apiVersion: knativekafka.kyma-project.io/v1alpha1
+  knative-eventing.channel.default.kind: KafkaChannel
+EOF
+```
+
+In this example the default channel is set to Kafka.
+
+### Kafka
+
+There is a Knative compatible [Kafka channel implementation](https://github.com/kyma-incubator/knative-kafka) which can be used for more production-ready eventing workloads but is still in alpha state.
+
+>**NOTE:** Kafka Channel integration is in alpha version. Use it only for testing purposes.
+
+The Knative channel implementation supports [these providers](https://github.com/kyma-incubator/knative-kafka/blob/9eb3fa3f6e67ffc80b162d2ef4c8a8a3942d9c5f/resources/README.md#kafka-providers):
+
+1. [Azure Event Hubs](https://azure.microsoft.com/en-us/services/event-hubs/)
+2. [Confluent Cloud](https://www.confluent.io/confluent-cloud)
+3. [Standard Kafka installation with no special authorization required](https://kafka.apache.org/quickstart)
+
+Please follow any of the links above on how to setup a Kafka cluster.
+
+Before starting the Kyma installation, the connection between Kyma and the Kafka cluster needs to be configured. This can be done via an installation override like this:
+
+```bash
+$ export $kafkaBrokers=<todo user>
+$ export $kafkaNamespace=<todo user>
+$ export $kafkaPassword=<todo user>
+$ export $kafkaUsername=<todo user>
+$ export $kafkaProvider=<local|azure|confluent>
+
+$ cat << EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: knative-kafka-overrides
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    component: knative-eventing-channel-kafka
+    kyma-project.io/installation: ""
+type: Opaque    
+stringData:
+  kafka.brokers: $kafkaBrokers
+  kafka.namespace: $kafkaNamespace
+  kafka.password: $kafkaPassword
+  kafka.username: $kafkaUsername
+  kafka.secretName: knative-kafka
+  environment.kafkaProvider: $kafkaProvider
+EOF
+```
+
+>**NOTE:** For other options, check this [link](https://github.com/kyma-incubator/knative-kafka/blob/master/resources/knative-kafka/values.yaml).
+
+Now that the installation has been customized, the Kyma installation can be triggered. 
+You can install Kyma with a custom component (`knative-eventing-channel-kafka` for the component and `knative-eventing-channel-kafka-test` for the TestDefinition) by following these [instructions](/root/kyma/#configuration-custom-component-installation).
```

Comment:
```suggestion
You can install Kyma with a `knative-eventing-channel-kafka` custom component by following these [instructions](/root/kyma/#configuration-custom-component-installation). The TestDefinition for the component is `knative-eventing-channel-kafka-test`.
```
Question: if we install the component, the override is not necessary? How can the component be used?

Reply:
The component can't be used without an installer override, because the kafka channel controllers needs to know how to connect to the Azure Eventhubs Namespace. The secret for the controller could also be created after the installation. This would require a restart of the controller however. We could also document this approach if you like. Let me know.